django/db/migrations/operations/base.py

```diff
@@ -85,6 +98,15 @@ def describe(self):
         """
         return "%s: %s" % (self.__class__.__name__, self._constructor_args)
 
+    def formatted_description(self):
+        """
+        Output the describe output in a more informative way.
+        """
+        description = self.describe()
+        if self.category is None:
+            return description
```

Comment:
I think we should display `?` when `category` is `None`, as [suggested](https://forum.djangoproject.com/t/makemigrations-highlighter-interesting-or-noise/24039/14) by Simon and Adam.

Reply:
Therefore, the base `Operation` class can be set as MIXED instead of `None` to be used in custom operations, as the only place that its category is set to `None`. Other operations have explicit operation categories. If one tries to set its custom category to `None` (instead of the default which is now `?`), I feel we shouldn't stop them.